testing/uclibcxx_test/Makefile

```diff
@@ -21,7 +21,9 @@
 include $(APPDIR)/Make.defs
 
 CXXEXT := .cpp
-ORIGS  := $(wildcard $(TOPDIR)/libs/libxx/uClibc++/tests/*.cpp)
+
+TESTDIR = $(TOPDIR)/libs/libxx/uClibc++/tests
```

Comment:
Looks good except the commit message does not reflect the actual change. I think it can be updated to: add TESTDIR variable or something like this